### PR TITLE
Added pull request template and issue template

### PR DESCRIPTION
**Pull request by iamrajiv**
Fixes: #589

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/proposal.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (eldadfux) [inline .github/ISSUE_TEMPLATE/proposal.md:1]: I think we could remove this template and just stay with the feature template.

**Comment by iamrajiv** [comment]
@eldadfux Done with the requested changes please review.

**Comment by eldadfux** [comment]
Gonna wait a few days for more feedback from the community on this PR. If everything goes well, I'll merge it.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+labels: "bug"
+---
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is)
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
```
Comment (christyjacob4) [inline .github/ISSUE_TEMPLATE/bug.md:11]: I feel CONTRIBUTING.md may not be so relevant for a bug report . You could skip this section.
```suggestion
## Description 
(A clear and concise description of what the bug is)
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  Don't only say "it doesn't work"!
+-->
+
+(Write what happened. Add screenshots, if applicable.)
+
+## Your Environment
+
+<!-- Include as many relevant details about the environment you experienced the bug in -->
+
+(Write Environment, Operating system and version etc.)
```
Comment (christyjacob4) [inline .github/ISSUE_TEMPLATE/bug.md:40]: You could add this check and probably add it to all the issue templates

### Have you spent some time to check if this issue has been raised before?

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+name: 📚 Documentation
+about: Report an issue related to documentation
+labels: "documentation"
+---
+
+## 📚 Documentation
+
+(A clear and concise description of what the issue is.)
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
```
Comment (christyjacob4) [inline .github/ISSUE_TEMPLATE/documentation.md:11]: You could replace this with the code of conduct.

File: .github/ISSUE_TEMPLATE/feature.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
+
+(Write your answer here.)
+
+## Motivation
+
+(Please outline the motivation for the proposal.)
+
+## Pitch
```
Comment (christyjacob4) [inline .github/ISSUE_TEMPLATE/feature.md:19]: I think motivation and pitch are two very broad terms. Would be better if you could replace them with more specific questions and also replace contributing guidelines with the code of conduct

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+Help us understand your motivation by explaining why you decided to make this change.
+
+You can learn more about contributing to appwrite here: https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md
+
+Happy contributing!
+
+-->
+
+## Motivation
```
Comment (christyjacob4) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
## What does this PR do ?
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+(Write your motivation here.)
```
Comment (christyjacob4) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
Provide a description of what this PR does
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
```
Comment (christyjacob4) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: You could move the contributing guidelines as the last item in this list. We would want the contributor to talk about the PR first.

**Comment by christyjacob4** [comment]
Rest of it LGTM 👍

**Review by Kolkies — APPROVED** [review]

**Review by PineappleIOnic — APPROVED** [review]

**Comment by iamrajiv** [comment]
@Kolkies Can you add Hacktoberfest label to it?

**Review by TorstenDittmann — APPROVED** [review]
This looks good to me 👍

**Comment by eldadfux** [comment]
THANK YOU @iamrajiv! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Comment by iamrajiv** [comment]
Thank You @eldadfux and other reviewers.

**Pull request merged**